dsa-solutions/lc-solutions/0000-0099/0083-remove-duplicates-from-sorted-list.md

```diff
@@ -1,121 +1,119 @@
 ---
 id: remove-duplicates-from-sorted-list
-title: Remove Duplicates from Sorted List Solution
-sidebar_label: 0083 Remove Duplicates from Sorted List
-tags:
-  - Linked List
-  - Two Pointers
-  - LeetCode
-  - Python
-  - Java
-  - C++
-description: "This is a solution to the Remove Duplicates from Sorted List problem on LeetCode."
+title: Remove Duplicates from Sorted List (LeetCode)
+difficulty: Easy
```

Comment:
delete `difficulty: Easy` and why you use it, because it is not a part of our file structure

Reply:
@thevijayshankersharma Make sure you need to work on new problem can't be work on existing problem solution. So, before write code you need update your code from our latest main branch and checkout our site if exist or not and if not available then try to work on it. 

***Additionally, please close these pull requests as they may cause issues with deployment and maintenance.***